src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -160,6 +160,19 @@ def load_arguments(self, _):
     with self.argument_context('aks upgrade') as c:
         c.argument('kubernetes_version', completer=get_k8s_upgrades_completion_list)
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+    
+    with self.argument_context('aks maintenanceconfiguration') as c:
+        c.argument('cluster_name', type=str, help='The cluster name.')
+
+    for scope in ['aks maintenanceconfiguration add', 'aks maintenanceconfiguration update']:
+        with self.argument_context(scope) as c:
+            c.argument('config_name', type=str, options_list=['--config-name'], help='The config name.')
+            c.argument('time_in_week', type=str, options_list=['--time-in-week'], help='Weekly schedule', required=False)
+            c.argument('not_allowed_time', type=str, options_list=['--not-allowed-time'], help='Not allowed time', required=False)
```

Comment:
it will be very helpful to user to show what's format of the param value, and avoid usage error, it's not straightforward from the descripiton now

Reply:
Add more info in help file.